### DISCUSSION TITLE
Adding currencies

### DISCUSSION THREAD

**Discussion by BorisFromBE** [openkfw/TruBudget#1145] [Ideas]
To make sure all partners of Burkina Faso  outside of Europe can use SIGFE in the future, we would like to ensure several foreign currencies are usable in SIGFE. It should be noted that both SIGFE and TruBudget should be able to at least display all currencies listed: 

- CAD, 
- MAD,
-  SAR, 
- QAR,
- KWD, 
- INR, 
- CNY, 
- JPY, 
- GBP, 
- CHF,
- SEK,
- NOK.

**Comment by Stezido** [comment]
true, I think it shouldn't be a problem to add and support all currencies you have listed or the currencies ECB is providing via their open API.
I transform it into an issue so the team can work on it next sprint

**Comment by georgimld** [comment]
This was implmenented in #1426, so this discussion will be closed
